lodash/pick throwing ERR_MODULE_NOT_FOUND when using @rjsf/shadcn

### Prerequisites

- [x] I have searched the existing issues
- [x] I understand that providing a [SSCCE](http://sscce.org/) example is tremendously useful to the maintainers.
- [x] I have read the [documentation](https://rjsf-team.github.io/react-jsonschema-form/docs)
- [x] Ideally, I'm providing a [sample JSFiddle](https://jsfiddle.net), [Codesandbox.io](https://codesandbox.io) or preferably a [shared playground link](https://rjsf-team.github.io/react-jsonschema-form/) demonstrating the issue.

### What theme are you using?

shadcn

### Version

6.0.0-beta.11

### Current Behavior

I followed the installation guide for the shadcn theme and ended up installing the following packages:

``` 
"@rjsf/core": "^6.0.0-beta.11",
"@rjsf/shadcn": "^6.0.0-beta.10",
"@rjsf/utils": "^6.0.0-beta.11",
"@rjsf/validator-ajv8": "^6.0.0-beta.11",
```

But when I try to run my project containing the following statement `import Form from "@rjsf/shadcn";` I get the following error:

``` 
 ⨯ Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/node_modules/lodash/pick' imported from /node_modules/@rjsf/shadcn/lib/RangeWidget/RangeWidget.js
Did you mean to import "lodash/pick.js"?
```

### Expected Behavior

_No response_

### Steps To Reproduce

I am installing `@rjsf/shadcn` in an already big project using Nextjs 13, so I'm not sure how to give a project setup to reproduce the issue 😔

### Environment

```markdown
- OS: macOS 15.5
- Node: 24.1.0
- npm: 11.3.0
```

### Anything else?

_No response_

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -16,6 +16,12 @@ should change the heading of the (upcoming) version to include a major version b
 
 -->
 
+# 6.0.0-beta.12
+
+## @rjsf/shadcn
+
+- Updated the building of `shadcn` to use the `lodashReplacer` with `tsc-alias` fixing [#4678](https://github.com/rjsf-team/react-jsonschema-form/issues/4678)
+
 # 6.0.0-beta.11
 
 ## @rjsf/antd

--- a/packages/shadcn/package.json
+++ b/packages/shadcn/package.json
@@ -44,7 +44,8 @@
   },
   "scripts": {
     "build:css": "tsx build-css.ts",
-    "build:ts": "tsc -b tsconfig.build.json && tsc-alias -p tsconfig.build.json",
+    "compileReplacer": "tsc -p tsconfig.replacer.json && move-file lodashReplacer.js lodashReplacer.cjs",
+    "build:ts": "npm run compileReplacer && rimraf ./lib && tsc -b tsconfig.build.json && tsc-alias -p tsconfig.build.json",
     "build:cjs": "esbuild ./src/index.ts --bundle --outfile=dist/index.js --sourcemap --packages=external --format=cjs",
     "build:esm": "esbuild ./src/index.ts --bundle --outfile=dist/rjsf-shadcn.esm.js --sourcemap --packages=external --format=esm",
     "build:umd": "rollup dist/rjsf-shadcn.esm.js --format=umd --file=dist/rjsf-shadcn.umd.js --name=@rjsf/rjsf-shadcn",

--- a/packages/shadcn/tsconfig.build.json
+++ b/packages/shadcn/tsconfig.build.json
@@ -11,6 +11,12 @@
   ],
   "tsc-alias": {
     "resolveFullPaths": true,
-    "verbose": true
+    "verbose": true,
+    "replacers": {
+      "lodash": {
+        "enabled": true,
+        "file": "lodashReplacer.cjs"
+      }
+    }
   }
 }

--- a/packages/shadcn/tsconfig.replacer.json
+++ b/packages/shadcn/tsconfig.replacer.json
@@ -1,0 +1,15 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es2017",
+    "outDir": "./",
+    "skipLibCheck": true,
+  },
+  "files": [
+    "../../tsc-alias-replacer/lodashReplacer.ts"
+  ],
+  "exclude": [
+    "./src",
+    "./test"
+  ]
+}

